cmd/container-manager/manage_ctr_mgr.sh

```diff
@@ -188,6 +188,13 @@ remove_docker(){
 	if [ -z "$pkg_name" ]; then
 		die "Docker not found in this system"
 	else
+		# Check docker version at versions.yaml from the runtime
+		docker_version=$(get_version "externals.docker.version" | cut -d '-' -f1 | cut -d 'v' -f2)
+		# Do not remove docker if the version is equal to the one
+		# that is supported
+		install_docker_version=$(get_docker_version)
+		[ "$docker_version" == "$install_docker_version" ] && return
```

Comment:
This shouldn't be part of the remove_docker as it may be the case that a user just want to remove the package even if it is the version that we support. I'll prefer to add this check inside the `install_docker()` 
Btw, this is also being checked on https://github.com/kata-containers/tests/blob/master/.ci/setup.sh#L51-L59